### DISCUSSION TITLE
I want you to have the option to show blocked sites from the beginning by default.

### DISCUSSION THREAD

**Discussion by 0rodna1** [iorate/ublacklist#756] [Ideas]
## Why

When subscribing to public rulesets, for those who are subscribing, rules that hit the wrong explosion may be added.
I want to avoid the situation of not recognizing the false explosion.
At present, the default is only "Hide", and you have to tap "Show" to show.
I want you to have the option to show blocked sites from the beginning by default.

## Expected Behavior

![image](https://github.com/user-attachments/assets/b2eedfc9-9887-4547-8d44-61d3589bd9be)

## Actual Behavior

![image](https://github.com/user-attachments/assets/063eb833-28f6-4528-aa94-6ecda4a6c5ee)

## Specifications

- Browser: Safari
- Other extensions/add-ons: AdGuard Pro

- Search engine: Google
- Language: Japanese
- Region: Japan

**Comment by LiquidSevens** [comment]
This would be a good feature. I can't personally vet every single domain that my filter lists have blocked. Occasionally, something I want to see does slip through the cracks.

Someone else said it best on another discussion (#623): "if i don't find what i'm looking for I might want to broaden my search to include the blocked sites as well ... if none of my preferred sites have an article on a particular topic i could turn to lower quality journals."

The hide-by-default behavior is great for the average use-case, but if I'm looking for something specific, it would be nice to have everything displayed by default ("bad" sites are still marked, after all).

As an alternative, I was hoping that someone had made a simple userscript that accomplishes this, but I've found nothing so far.
